subjects/frontback/README.md

```diff
@@ -0,0 +1,26 @@
+## frontback
+
+### Instructions
+
+Write a program that prints the previous letter and the next letter of the alphabet.
```

Comment:
LGTM, except that this doesn't mention explicitly only the Latin alphabet!

Reply:
fixed.